src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -353,6 +353,16 @@ public bool Equals(NuGetFramework other)
             return Comparer.Equals(this, other);
         }
 
+        public static bool operator ==(NuGetFramework left, NuGetFramework right)
+        {
+            return Comparer.Equals(left, right);
+        }
+
+        public static bool operator !=(NuGetFramework left, NuGetFramework right)
+        {
+            return !(left == right);
```

Comment:
super-nit: wouldn't negating `Comparer.Equals` be more consistent with the other implementations and have one fewer method invocations?